CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the LaunchDarkly's EventSource implementation for C# will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org).
 
+## [3.2.1] - 2019-01-07
```

Comment:
does the date need to be updated?

Reply:
Yes, but I'll update it when I actually do a release.